apps/projects/app/components/App/App.js

```diff
@@ -219,6 +223,88 @@ class App extends React.PureComponent {
     }))
   }
 
+  onSubmitBountyAllocation = async issues => {
+    console.log('repos:', this.props.repos)
+    console.log('bounty allocation submitted', issues)
+    let bountySymbol = this.props.bountySettings.bountyCurrency
+    let bountyToken
+    this.props.tokens.forEach(
```

Comment:
why not use map here?

Reply:
Bad form to use map if you arn't returning an array.